src/v/cloud_storage/partition_probe.cc

```diff
@@ -94,15 +95,17 @@ partition_probe::partition_probe(const model::ntp& ntp) {
           [this] { return _spillover_mat_latency.public_histogram_logform(); },
           sm::description(
             "Spillover manifest materialization latency histogram"),
-          labels),
+          labels)
+          .aggregate({partition_label}),
```

Comment:
If we unconditionally aggregate is there a way we can share the histogram to avoid the cpu overhead alltogether?

Maybe an optimization for later.

Reply:
Actually just forgot to add the conditional aggregation... Adding it now.